google/cloud/pubsub/samples/samples.cc

```diff
@@ -374,6 +374,33 @@ void CreatePushSubscription(
   (std::move(client), argv.at(0), argv.at(1), argv.at(2), argv.at(3));
 }
 
+void CreateUnwrappedPushSubscription(
+    google::cloud::pubsub::SubscriptionAdminClient client,
+    std::vector<std::string> const& argv) {
+  //! [START pubsub_create_unwrapped_push_subscription]
+  namespace pubsub = ::google::cloud::pubsub;
+  [](pubsub::SubscriptionAdminClient client, std::string const& project_id,
+     std::string const& topic_id, std::string const& subscription_id,
+     std::string const& endpoint) {
+    auto sub = client.CreateSubscription(
+        pubsub::Topic(project_id, std::move(topic_id)),
+        pubsub::Subscription(project_id, std::move(subscription_id)),
+        pubsub::SubscriptionBuilder{}.set_push_config(
+            pubsub::PushConfigBuilder{}.set_push_endpoint(endpoint).set_wrapper(
+                pubsub::PushConfigBuilder::MakeNoWrapper(true))));
+    if (sub.status().code() == google::cloud::StatusCode::kAlreadyExists) {
+      std::cout << "The subscription already exists\n";
+      return;
+    }
+    if (!sub) throw std::move(sub).status();
```

Comment:
Ditto on testing `if (!sub)` first.

Reply:
FWIW, it looks like this slipped through the cracks, so the two samples now look a little different.